service/resource/clusterrolebindingv2/create.go

```diff
@@ -0,0 +1,62 @@
+package clusterrolebindingv2
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	apiv1 "k8s.io/api/rbac/v1beta1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
```

Comment:
For the future it would be even more easy to just provide the skeleton of a resource and provide the implementations separately. That way we can break it down even more. The registration should then be the last thing. You can check the last PRs in the cert-operator. That shows what I mean.